packages/datadog-plugin-http/test/client.spec.js

```diff
@@ -1133,6 +1133,34 @@ describe('Plugin', () => {
             })
           })
         })
+
+        it('should support adding request headers when header set as an array', done => {
+          const app = express()
+
+          app.get('/user', (req, res) => {
+            res.status(200).send()
+          })
+
+          getPort().then(port => {
+            agent
+              .use(traces => {
+                const meta = traces[0][0].meta
+
+                expect(meta).to.have.property(`${HTTP_REQUEST_HEADERS}.x-foo`, `bar`)
+              })
+              .then(done)
+              .catch(done)
+
+            appListener = server(app, port, () => {
+              const req = http.request(`${protocol}://localhost:${port}/user`, res => {
+                res.on('data', () => { })
+              })
+
+              req.setHeader('x-foo', ['bar'])
```

Comment:
is it possible to set an array with multiple values? Such as
```javascript
              req.setHeader('x-foo', ['bar', 'baz'])
```
if it is, does it make sense to have the test do that? So that we can assert on `bar,baz`, which is a more complete example

Reply:
Done